roles/docker/tasks/package_docker.yml

```diff
@@ -102,6 +102,19 @@
   notify:
   - restart docker
 
+- stat: path=/etc/sysconfig/docker-network
+  register: sysconfig_docker_network_check
+
+- name: Configure Docker Network OPTIONS
+  lineinfile:
+    dest: /etc/sysconfig/docker-network
+    regexp: '^DOCKER_NETWORK_OPTIONS=.*$'
+    line: "DOCKER_NETWORK_OPTIONS='\
+      {% if openshift.node is defined and openshift.node.sdn_mtu is defined %} --mtu={{ openshift.node.sdn_mtu }}{% endif %}'"
```

Comment:
I was going to complain that this blows away any pre-existing DOCKER_NETWORK_OPTIONS but I guess that matches the behavior for OPTIONS, so ok